locales/en-US/strings.properties

```diff
@@ -54,6 +55,7 @@ confirm_history_delete_p1=Are you sure you want to delete every instance of this
 # page from history.
 confirm_history_delete_notice_p2=This action cannot be undone.
 menu_action_save_to_pocket=Save to Pocket
+menu_action_remove_pocket=Remove from Pocket List
```

Comment:
Probably confirm with pocket(?) about this string. In particular the opposite string is just "Save to Pocket" (no "List")

Reply:
Designers said "Delete on Pocket" and "Archive on Pocket"